app/Http/Controllers/Service/Admin/VouchersController.php

```diff
@@ -22,7 +25,7 @@ class VouchersController extends Controller
      */
     public function index()
     {
-        $vouchers = Voucher::all();
+        $vouchers = DB::table('vouchers')->orderBy('id', 'desc')->paginate(50);
```

Comment:
I thought so too, but my ide was more confident about this. Mind you php storm can be funny sometimes...